flow/src/main/java/com/sequenceiq/flow/controller/FlowController.java

```diff
@@ -65,4 +66,16 @@ public FlowCheckResponse hasFlowRunningByChainId(String chainId) {
     public FlowCheckResponse hasFlowRunningByFlowId(String flowId) {
         return flowService.getFlowState(flowId);
     }
+
+    @Override
+    @AccountIdNotNeeded
+    public List<FlowLogResponse> getFlowLogStatusesByFlowIds(List<String> flowIds, int size, int page) {
+        return flowService.getFlowLogsByIds(flowIds, PageRequest.of(page, size));
```

Comment:
this way size and page is required, and size should not be 0
either please add validation to the endpoint, or initialize page object only if they are set